src/jobs/ega/egaPermissionsReconciliation.ts

```diff
@@ -39,12 +41,50 @@ const JOB_NAME = 'RECONCILE_EGA_PERMISSIONS';
  * 3) Retrieve corresponding list of users from EGA API
  * 4) Create permissions, on each dataset, for each user on the DACO approved list, if no existing permission is found
  * 5) Process existing permissions for each dataset + revoke those which belong to users not on the DACO approved list
+ * 6) Return completed JobReport
+ * @returns Promise<JobReport<ReconciliationJobReport>>
+ * @example
```

Comment:
added this example report to show how the completionStatus lines up with the `details` from each of the create permissions/delete permissions process steps